docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -1,66 +1,49 @@
+* The {OpenShift} ({OpenShiftShort}) nodes where the `manila-share` service is to be deployed can reach the management network that the storage system is in.
+* If the deployment uses CephFS through NFS as a storage back end, a new clustered Ceph NFS service is deployed on the {CephCluster} cluster with the help
+of Ceph orchestrator. For more information, see xref:creating-a-ceph-nfs-cluster_migrating-databases[Creating a Ceph NFS cluster].
+* Services such as the {identity_service_first_ref} and memcached are available prior to adopting the Shared File Systems services.
+* If you enabled tenant-driven networking by setting `driver_handles_share_servers=True`, the {networking_first_ref} is deployed.
 
 .Procedure
 ifeval::["{build}" != "downstream"]
 . Define the `CONTROLLER1_SSH` environment variable, if it link:stop_openstack_services.md#variables[hasn't been
 defined] already. Then copy the configuration file from {OpenStackShort} {rhos_prev_ver} for reference.
 endif::[]
 ifeval::["{build}" != "upstream"]
-. Define the `CONTROLLER1_SSH` environment variable, if it hasn't been
+. Define the `CONTROLLER1_SSH` environment variable, if it has not been
 defined already. Then copy the configuration file from {OpenStackShort} {rhos_prev_ver} for reference.
 endif::[]
 +
 ----
-$CONTROLLER1_SSH cat /var/lib/config-data/puppet-generated/manila/etc/manila/manila.conf | awk '!/^ *#/ && NF' > ~/manila.conf
+$ CONTROLLER1_SSH cat /var/lib/config-data/puppet-generated/manila/etc/manila/manila.conf | awk '!/^ *#/ && NF' > ~/manila.conf
 ----
 
-. Review this configuration alongside any configuration changes that were noted since {OpenStackShort} {rhos_prev_ver}. Not all of it makes sense to bring into the new cloud environment:
-// - TODO link config diff tables for RHOSP 17.1 (Wallaby) to RHOSP 18 (Antelope) -
-//kgilliga: The following list might need to be placed in its own module and linked here. This is too much text for a procedure.
-* The {rhos_component_storage_file} operator is capable of setting up database related configuration
-(`[database]`), service authentication (`auth_strategy`,
-`[keystone_authtoken]`), message bus configuration
-(`transport_url`, `control_exchange`), the default paste config
-(`api_paste_config`) and inter-service communication configuration (
-`[neutron]`, `[nova]`, `[cinder]`, `[glance]` `[oslo_messaging_*]`). So
-all of these can be ignored.
-* Ignore the `osapi_share_listen` configuration. In {rhos_long} {rhos_curr_ver}, you rely on
-{OpenShift} routes and ingress.
-* Pay attention to policy overrides. In {rhos_acro} {rhos_curr_ver}, the {rhos_component_storage_file} ships with a secure
-default RBAC, and overrides may not be necessary.
+. Review the configuration file for any configuration changes that were made since {OpenStackShort} {rhos_prev_ver}. Not all of it makes sense to bring into the new cloud environment:
```

Comment:
Step 2 (Review the configuration file) has a lot of reference information. It’s too much for a step. I’m considering moving the bullet to a separate reference topic that we can link to. It could be called something like “Preparing the Shared File Systems service configuration”.  WDYT?

Reply:
Sounds good to me.